tests/amd_migraphx_test/gdf/out.f32

```diff
@@ -0,0 +1 @@
+�pl�H�?|ɓ���5+iA3�%��\i���@��o@�BA
```

Comment:
do we need to keep this file in the repo? is this supposed to be a golden result? if so, for which digit?

Reply:
@asalmanp Pavel likes to keep a golden result. I can change the name to reflect it. Do we not want to keep this at all?

Reply:
we can keep it but it needs to have a unique name, and please also mention it in the readme that this is a golden result for a specific digit.